src/analysis-format.ts

```diff
@@ -140,7 +140,7 @@ export interface ElementLike extends Feature {
    *
    * e.g. `['demos/index.html', 'demos/extended.html']`
    */
-  demos: string[];
+  demos: Demo[];
```

Comment:
I think demos should be as standard as descriptions. They're just named links, and they're really useful for all UI-related stuff.